src/Symfony/Bridge/Doctrine/Tests/Security/User/EntityUserProviderTest.php

```diff
@@ -156,7 +157,7 @@ public function testLoadUserByUserNameShouldLoadUserWhenProperInterfaceProvided(
             ->method('loadUserByUsername')
             ->with('name')
             ->willReturn(
-                $this->getMockBuilder('\Symfony\Component\Security\Core\User\UserInterface')->getMock()
+                $this->getMockBuilder(UserInterface::class)->getMock()
```

Comment:
please revert all the non required ::class, we favor helping future merges than doing these CS updates